.github/workflows/ansible_lint.yml

```
@@ -0,0 +1,20 @@
+---
+
+name: ansible tests
+
+on:  # yamllint disable-line rule:truthy
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install dependencies
+        run: pip3 install -r .dev_requirements.txt
+
+      - name: test playbook
+        run: molecule test --driver-name docker
```

Please remove `--driver-name` parameter. Using `containers` driver will automatically detect the driver.

```suggestion
        run: molecule test
```